EIPS/eip-6366.md

```diff
@@ -44,6 +44,8 @@ interface IERC6366Core {
 
   function permissionRequire(uint256 _required, uint256 _permission) external view returns (bool isPermission);
 
+  function hasSufficientPermission(address _actor, address _owner, uint256 _required) external view returns (bool hasPermission);
```

Comment:
I'm prefer a short name like `hasPermission()`

Reply:
Yes you're right shorter is clearer !